src/components/auth-module/register-view/register-view.js

```diff
@@ -0,0 +1,209 @@
+import React, { Component } from 'react';
+import { translate } from 'react-i18next';
+import { translateOptions } from '../../../i18n/config';
+import { Button, Form } from 'semantic-ui-react';
+import { Link } from 'react-router-dom';
+
+import ErrorMessageView from '../../common/error-message-view';
+import { isValidEmail } from '../../../utils/validation-utils';
+import RegisterMessageView from './register-message/register-message';
+
+class RegisterView extends Component {
+  /***************************
+   *        CONSTRUCTOR
+   ***************************/
+  constructor(props) {
+    super(props);
+
+    this.state={
+      fields: {
+        email: '',
```

Comment:
`companyName` and `password` is missing state. Make sure we have atleast empty string validations for all the keys.